### PR TITLE
Add tile

### PR DESCRIPTION
**Pull request by Vimalakirti**
The PR added the tile layer.

### CODE REVIEW

File: src/layer/tile.rs [hunk 1]
```diff
@@ -0,0 +1,61 @@
+use crate::basic_block::*;
```
Comment (ddkang) [inline src/layer/tile.rs:1]: Please describe the logic for this file

Comment (Vimalakirti) [reply src/layer/tile.rs:1]: added

**Review by liliatangxy — APPROVED** [review]

**Comment by ddkang** [comment]
As long as there are no security issues, lgtm

**Comment by Vimalakirti** [comment]
> As long as there are no security issues, lgtm

There's no security issue in this PR.

**Review by ddkang — APPROVED** [review]

**Pull request merged**